src/scripts.js

```diff
@@ -28,6 +45,7 @@ cardArea.addEventListener('keyup', checkKeyPressed);
 searchButton.addEventListener('click', searchRecipes);
 
 function searchRecipes() {
+  console.log(cookbook)
```

Comment:
Do we want this console log here? What purpose does it serve?

Reply:
I forgot to leave it out. I was just checking if the correct information is there. I will remove it.